examples/.env

```diff
@@ -52,6 +52,10 @@ TAVILY_API_KEY=
 # For using Brave (tool)
 BRAVE_API_KEY=
 
+# For using Firecrawl (tool)
+FIRECRAWL_HOST=
```

Comment:
ah, missed that one:
```
FIRECRAWL_HOST=https://api.firecrawl.dev
```

or we just skip the var here

Reply:
IMO, the var can be kept as you can run Firecrawl locally, having the var can help testing it locally without using cloud service 🤔

Reply:
but lets set the default url in the .env

Reply:
Done, the value has been moved to `.env`, easier for end users 👍🏻